File: app/src/main/java/com/example/coronamap/VO/CoronaMapModel.java
package com.example.coronamap.VO;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.Required;

public class CoronaMapModel extends RealmObject {

    @PrimaryKey
    private int id;

    @Required
    private String locationX;
    @Required
    private String locationY;

    private String keyword;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getLocationX() {
        return locationX;
    }

    public void setLocationX(String locationX) {
        this.locationX = locationX;
    }

    public String getLocationY() {
        return locationY;
    }

    public void setLocationY(String locationY) {
        this.locationY = locationY;
    }

    public String getKeyword() {
        return keyword;
    }

    public void setKeyword(String keyword) {
        this.keyword = keyword;
    }
}

File: app/src/main/java/com/example/coronamap/model/CoronaMapModel.kt
package com.example.coronamap.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import io.realm.annotations.Required

open class CoronaMapModel : RealmObject() {

    @PrimaryKey
    var id: Int = 0

    @Required
    var locationX: Double? = null
    @Required
    var locationY: Double? = null

    var keyword: String? = null
    var cityName: String? = null
}
